src/lib/components/Communities/CommunityHeader.js

```diff
@@ -32,8 +34,11 @@ class CommunityHeaderInfo extends React.Component {
     const logoLink = community ? community?.links?.logo : imagePlaceholderLink;
     return (
       <Container className="community-header">
-        <Image size="mini" src={logoLink} /> {community?.metadata?.title}
-        <CommunityHeaderButton />
+        <Image className="community-logo-header" src={logoLink} />
```

Comment:
you miss the fallback url e.g point to `/static/images/sqaure-placeholder.png`?

Reply:
Right, I was actually checking for the fallback above with:
```
const logoLink = community ? community?.links?.logo : imagePlaceholderLink;
```
But is not anymore needed.